src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,284 @@
+## Reduce data ingested by setting `global.lowDataMode` [#reduce-data-ingest]
 
-```
-- kube_
-- container_
-- machine_
-- cadvisor_
-```
+If you're looking to cut costs, you can modify the `global.lowDataMode` parameter value. This will cut down on the number of metrics you send. The modification of this parameter will depend on the method you used to install Kubernetes:
 
-### New Relic Logging
-
-If `lowDataMode` is enabled, Labels and Annotations are not included in the logs forwarded to New Relic. This reduces the overall data ingest into New Relic.
-
-The following fields are retained: `cluster_name`, `container_name`, `namespace_name`, `pod_name`, `stream`, `message` and `log`.
-
-<DNT>**Low Data Mode Log Example**</DNT>
-
-The following log record contains all the attributes typically captured by the [New Relic Logs Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/):
-
-```json
-[
-  {
-    "cluster_name": "api-test",
-    "kubernetes": {
-      "annotations": {
-        "kubernetes.io/psp": "eks.privileged"
-      },
-      "container_hash": "fryckbos/test@sha256:5b098eaf3c7d5b3585eb10cebee63665b6208bea31ef31a3f0856c5ffdda644b",
-      "container_image": "fryckbos/test:latest",
-      "container_name": "newrelic-logging",
-      "docker_id": "134e1daf63761baa15e035b08b7aea04518a0f0e50af4215131a50c6a379a072",
-      "host": "ip-192-168-17-123.ec2.internal",
-      "labels": {
-        "app": "newrelic-logging",
-        "app.kubernetes.io/name": "newrelic-logging",
-        "controller-revision-hash": "84db95db86",
-        "pod-template-generation": "1",
-        "release": "nri-bundle"
-      },
-      "namespace_name": "nrlogs",
-      "pod_id": "54556e3e-719c-46b5-af69-020b75d69bf1",
-      "pod_name": "nri-bundle-newrelic-logging-jxnbj"
-    },
-    "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
-    "plugin": {
-      "source": "kubernetes",
-      "type": "fluent-bit",
-      "version": "1.8.1"
-    },
-    "stream": "stderr",
-    "time": "2021-09-14T12:30:49.138824971Z",
-    "timestamp": 1631622649138
-  }
-]
-```
+    * **Guided install** (New Relic CLI, Helm, manifest). The installation command includes `lowDataMode` by default because it has these parameters: `NRI_CLI_LOW_DATA_MODE`, `NRI_CLI_PROMETHEUS_AGENT_LOW_DATA_MODE`, and `NRI_CLI_LOGGING_LOW_DATA_MODE`.
+
+    * **Helm**. Set the `global.lowDataMode` parameter in the [`nri-bundle` chart](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) to true.
+
+    * **Manifest**. `LowDataMode` is only for Helm charts, and Helms charts are templates applied to manifests. When someone applies `LowDataMode` in the Helm chart, it makes a few small changes to the final manifest. The changes in the manifests are about specific settings, but not for a parameter named `LowDataMode`.
```

Comment:
@nbaenam I'm still unable to understand this sentence. If you think more knowledgeable readers will know what this means, I'll defer.

Reply:
I think it's clear for K8s users, but thank you for your comment.